include/sys/fs/zfs.h

```diff
@@ -896,6 +896,7 @@ typedef enum mmp_state {
 typedef enum pool_scan_func {
 	POOL_SCAN_NONE,
 	POOL_SCAN_SCRUB,
+	POOL_SCAN_RESCRUB,
 	POOL_SCAN_RESILVER,
```

Comment:
I think that changing the value of `POOL_SCAN_RESILVER` will break user/kernel compatibility (i.e. new userland with old kernel will get confused), due to `pool_scan_stat_t:pss_func`, and also break on-disk compatibility due to `dsl_scan_phys_t:scn_func`.  I think you can safely add the new value at the end (just before POOL_SCAN_FUNCS).